src/app/screens/confirmStxTransaction/index.tsx

```diff
@@ -137,7 +148,7 @@ function ConfirmStxTransaction() {
     const modifiedMemoString = txMemo.content.split('\u0000').join('');
 
     setAmount(txAmount);
-    setStateFee(txFee);
+    if (!fee) setStateFee(txFee);
```

Comment:
This if will never fire. fee will always be an instance of BigNumber and a class instance is always truthy.